early-attack-detector-py/src/constants.py

```diff
@@ -33,4 +33,7 @@
 EXTRA_TIME_BOTS = [
     '0xa91a31df513afff32b9d85a2c2b7e786fdd681b3cdd8d93d6074943ba31ae400'  # funding tornado cash
 ]
-EXTRA_TIME_DAYS = 180
\ No newline at end of file
+EXTRA_TIME_DAYS = 180
+
+ETH_BLOCKS_IN_ONE_DAY = 7200
+THREE_SECOND_BLOCKS_IN_ONE_DAY = 28800
```

Comment:
Could you explain this?

Reply:
Resolving, since we discussed this verbally.